eZ/Publish/Core/FieldType/User/UserStorage/Gateway/LegacyStorage.php

```diff
@@ -10,6 +10,8 @@
 
```

Comment:
AFAIR this class is not used, so you can safely drop it. It's also a part of separate ticket, but since you're re-implementing the whole thing, it can be also in the scope.

Reply:
Ok, I decided to not drop things which are not strictly part of task, but that's ok with me :)